vscode/src/memfs.ts

```diff
@@ -131,7 +131,11 @@ export async function initFileSystem(context: vscode.ExtensionContext) {
             log.debug("code from query: " + code);
             try {
               linkedCode = await compressedBase64ToCode(code);
-              const codeFile = vscode.Uri.parse(`${scheme}:/code.qs`);
```

Comment:
Can we wait with this? The code I'm sharing may be incorrect if I'm asking for help or providing wrong code intentionally. It may not have OPENQASM in it.